zipline/data/loader.py

```diff
@@ -172,6 +159,12 @@ def load_market_data(trading_day=None, trading_days=None, bm_symbol='SPY',
         now,
         environ,
     )
+
+    # combine dt indices and reindex using ffill then bfill
+    all_dt = br.index.union(tc.index)
+    br = br.reindex(all_dt, method='ffill').fillna(method='bfill')
+    tc = tc.reindex(all_dt, method='ffill').fillna(method='bfill')
```

Comment:
The tradeoff here that I'm concerned about is that we'll end up having benchmark returns for much further back than 5 years (which is the max amount we can get from IEX), except the returns of that benchmark will just be the same b/c we're `bfill`ing.

What is it that you're trying to get out of reindexing here?

Reply:
I'm trying to sync the two time series.  

The Federal Reserve H15 report is not released on a timely basis with market closes, so there will always be missing data on the front-end of the treasury curve time series.  At the same time, there are dates missing from both time series relative to each other.  Joining the time series, re-indexing, and then filling the missing data solves both issues.

If the concern is filling data back past 5 years on the benchmark, you could add a line to drop all dates older than the most recent start of the two respective time series.

Reply:
OK. That makes sense to me and sounds fair. Also just did some quick profiling and wasn't what I had thought it would be like.

Can you you `amend` your commit to use a [Commit Prefix](http://www.zipline.io/development-guidelines.html#commit-messages)? I think `MAINT:` would be appropriate here.

Reply:
@freddiev4 this PR is good to go